components/widgets/team_icon/team_icon.test.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import TeamIcon from './team_icon';
+
+describe('components/widgets/team-icon', () => {
+    test('should match the snapshot', () => {
```

Comment:
I miss some more tests here, like using a different `size`, or setting the `url` property, and the `withHover` property too.